newsFeed.html

```diff
@@ -19,14 +19,52 @@
6EyfJaozKcZE1ORCSGRp0pFuDKFCRcpyOdjFmLJoSK8ZEkWAJZTI/6XxLmk3keMgsnXxMnTLFiyWinozbVRvdttt+/EyS7tOJJPaOcvHNtciOzp+XVlDOVJKa2xjL5FuvLZn06/rsgkYmtWqdJ0+FuL/TFZX/BwN7Z8DaUMfJ6Dd121hJNd8GDU0qVSTlNqMcvpjYhllfgqolLwhYOcpen09XjbPY7ypSxTa7RINGoQhBKCxFbL3fU0mlh9ehTHjpX7Zm9nkPiFcVRrHJsxHRw8r7HSeJopVp423MKbRz43SoGRKyeFdySXL42GVJdhrct4A3TCo2RRjgmpS9UfiX7FerJgRqeuHxP9gVYk9GxZSxFHQ+DtOnSjVc+VSbnBf6Wc3R/SvhHoNkn5cMrDUUmuzwedzMjjGvs2NWyZgtBMZnlFgcASQbAkwmABaCBkwmAkROJKBIdAZFMhkyWZDNDoVkUmRhzQGCyEZ5Smw6XMAKDwz6ZvRyG5b3LSSRpW14+pztK4wW6d2ceTDfoopUdZQuMlylUOToajg0KOro4MnGfpFlNHVUapdpVTlKWsRLdLWo9zjnxp/RRTR1EahYhI5inrUO5Yp63HuRfHmvQ3ZHSwZLFnNLXIdw467HuD4sn0HtE2dTqqMqc89VFpdiao9jltQ1aM8fK/c6ahUUop90j6TiZZZIfsqZF0nohdHOc49jN1GePSjYqwZzmqSalheqUnhLq5MfMqRaDLem6hmUKUfdt+3NnQTl6G9kkmYFro/BB7vzJRalJdMrkjMndTt6MqU60q+W+Fv+qP+nPYONuMdmlT8GF4jkvMb9zna1SElhPdE2qVKk8tte6WdkZtCCTJwhSslOdss062OZbVcglFNEG62W6/IWkzdmi46meo0P8AuQ3ztLfuirGXX6NFq2Xrj7JgqicnZ0Wh0PMuKceifE/hbnoDkjx6trE6NWLpvEo7vs12Z2Fl4oVWCklh/qXZnlc3i5ZVJeB8U0rR1rqLuM6qOXlrTInrLOJcXIV7o6vjQDmu5yz1xkdTXJew64mQHdHVOSBckcl/H5ew38fkOuHMHdHWOSAc0clLX5APXpf3kb+HMHdHWTkiKUkcvLW5exFLWZd0OuJMVzR1E5IickcvLV59wHq0+6KLiSFczkRNjDM905w4yDjNkUQkAxOpsONVkMQkKMWIV2WIVpdyig4sVxRkaUK0u5PCrLuZtORYgyUoDWXo1pdwp3EkstmVWvFHZbsDzXJpN+77ZDHBfknPMorRr067ljP2PQdFulKlDD9UPSzzSMtjpPCup8M+F8pbb/5ujR0Sj0qhOPk7XZ3lxdcFPL3b2S6tvkkZmYUX5lRqdaX2gv8ALH/cqajqDjNZWeFuS/8ALGzObv8AV+KWXzz8Epz2dydI6q81lcLxjt9Tj9RvuKXXfKa5oCjczqbQi287vpgirafXbWYNY3Ebb8h2/BkVqjbSWzeft0IJ9+ucfKLtXTqmcvCfMqV7VrrkKaJuLIZ1cAeaPKiQSjhlEkxHZbcsosqqoqUn2wvkoRntgjuaucLt+4OtmctEdSo223ze5Ys7lwez+UUwoso0mqInRU9Qi9nJr5LOU+v5OYctizaXri8c12OeWD6HWT7Nx/LI5Y9xQqRkk0MyNUVB29xngeSAGMNlCb9x8ANDAExthNAMYATaBbQDYLDQplsEJgnQIEgkCh0YxJFhpkSZYhS2y9kKG6GTCRKqSSy3kidx2SNQvyL0TUqbCuKmI4QNrPO7ZHeMdQVWQlkbl1IKK5yfTZfJat117kEljhj23fyWKQy8izerLmdhqFZxmmny5DPkV6jKTVxojil1lZ2ErnzYwcpb8se//wAMW9gvM74x8GdZ3e+G8F+pT2T6/wB/k4JRaZ60ZJo7fSfKjSi44Tayyrqd/hNRay+mxyqvpR2TZapXfEm5LLxhe3uC3VFexU1C89XCu7aZnXN1z+SzfJPMsrOcYKFenj8MMUhG2RSqkLedx5rDAkVSJtjSkQ5CmwB0TbHyOgR0YAYw2RGMSwrNcnguUL9/q3M4NCuKZk2jcp1FJZQZiUqrXItK9fsReL6KLIvZfkwOIrK8WN0FSrqW3IHVh7IkciORN5YLpmCQMFsndEbyR0wGOxkJsSLEwkOCmWban+p8ly+TJWCTpWSUKPWX0QriplpB8RWctxmkQUnJ2yWrPJWyHNkQGNFFy1kSVVuitbPct1dx47RGepWVZP1MsUWQS33+4dKobwxpK4l7JXqyGVT3AlJDuVojGNMrzNfTL7KUZc0/usGRUIozaeVs0c8lZ243R10qSlJNcs/OEkQN8OemG/qslWz1Tii84i1hJd285Cuqucvq+n5ItHQmV6tfMsv+37FerPL+mB5ywmvt+f8Aggb/ANh0gWNV5/QiqSHlIikxkhGwWxIQhxRIcYcxhCEIABRDQ0R2zGFxD5ByIxghJjCMAu0blvbJLxy7mang0KM8xz/eSUo0Uix3KXcFyfcJgsCCzMYkLIiwo6RccuFJFWmt18hV3uFE57pEs5bESCjul8gzCJH6HkAEhMAw9N7l3OxQyW6Eth4PZLKvZHNYeUR+b3RPVKkwS0ww2iTzkN5yIRAsr0RI55BY8EScAAWkQFyF2+v3K7gu42yA4jqZYVTOF8hXLilFqSbecrs8lZSAmCguTCnIjyPwj8IwARDuIuExhkx8jqI+AGsEJIfAsmBY4IzFgxgkhxkOYwwhMRjDMtWM98d/3KjChLDTA1oKNbAHCBxguRKihmjiEWFJKP8AUh63MYQfQklsVF7j1OYhBFa2Mh4sQgWGh3EOnLAhGsWrRLJ5K9SPukIQzYsVRHhe7/A6a7DiFsrVj8Y2RCNZuqFwhcAwjWChcIsDCMYWB+EQjGaH4QWhCAahYEIRg0MMIRg0MOhCMah8jiEA1AsSEIJqBYhCMYvUp5SHYhC0Oj//2Q==" alt="flu shot pic" class="news-feed-pics"> 
```

Comment:
Add this image to your repo in order to make the link more manageable

Reply:
:shipit: